defopt.py

```diff
@@ -151,7 +153,7 @@ def _get_help_string(self, action):
                 info.append('type: %(type)s')
         if (not isinstance(action, _AppendAction)
                 and '%(default)' not in action.help
-                and action.default is not SUPPRESS
+                and action.default not in [SUPPRESS, _SUPPRESS_BOOL_DEFAULT]
```

Comment:
List membership is checked by equality, not identity. Given that the action defaults could be set to arbitrary objects it's probably worth being more specific here. You'd have to do `action.default is not SUPPRESS and action.default is not _SUPPRESS_BOOL_DEFAULT` or `all(action.default is not x for x in [SUPPRESS, _SUPPRESS_BOOL_DEFAULT])`.

Reply:
Yeah, was too lazy to do that :) fixing